clustered/client/src/main/java/org/ehcache/clustered/client/config/ClusteringServiceConfiguration.java

```diff
@@ -219,13 +220,25 @@ public ServerSideConfiguration getServerConfiguration() {
     return serverConfiguration;
   }
 
+  /**
+   * The timeouts for all cache operations
+   *
+   * @return the cache timeouts
+   */
+  public Timeouts getOperationTimeouts() {
+    return operationTimeouts;
+  }
+
   /**
    * The timeout for cache read operations.
    *
-   * @return the cache read operation timeout; may be {@code null} indicating the default timeout is used
+   * @return the cache read operation timeout
+   *
+   * @deprecated Use {@link #getOperationTimeouts()}
    */
-  public TimeoutDuration getReadOperationTimeout() {
```

Comment:
Again API breakage that we could easily prevent.

Reply:
Same answer :-)